src/Tickets/Emails/Settings.php

```diff
@@ -0,0 +1,296 @@
+<?php
+
+/**
+ * Tickets Emails Settings class
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Emails
+ *
+ */
+
+namespace TEC\Tickets\Emails;
+
+use TEC\Tickets\Commerce\Settings as CommerceSettings;
+use Tribe__Template;
+use Tribe__Tickets__Main;
+
+class Settings {
+
+	/**
+	 * The option key for email sender's name.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	static $option_sender_name = 'tec-tickets-emails-sender-name';
+
+	/**
+	 * The option key for email sender's email.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	static $option_sender_email = 'tec-tickets-emails-sender-email';
+
+	/**
+	 * The option key for the email header image url.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	static $option_header_image_url = 'tec-tickets-emails-header-image-url';
+
+	/**
+	 * The option key for the email header image alignment.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	static $option_header_image_alignment = 'tec-tickets-emails-header-image-alignment';
+
+	/**
+	 * The option key for the email header background color.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	static $option_header_bg_color = 'tec-tickets-emails-header-bg-color';
+
+	/**
+	 * The option key for the email ticket background color.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	static $option_ticket_bg_color = 'tec-tickets-emails-ticket-bg-color';
+
+	/**
+	 * The option key for the email footer content.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	static $option_footer_content = 'tec-tickets-emails-footer-content';
+
+	/**
+	 * The option key for the email footer credit.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	static $option_footer_credit = 'tec-tickets-emails-footer-credit';
+
+	/**
+	 * Gets the template instance used to setup the rendering html.
+	 *
+	 * @since TBD
+	 *
+	 * @return Tribe__Template
+	 */
+	public function get_template() {
+		if ( empty( $this->template ) ) {
+			$this->template = new Tribe__Template();
+			$this->template->set_template_origin( Tribe__Tickets__Main::instance() );
+			$this->template->set_template_folder( 'src/admin-views/settings/emails' );
+			$this->template->set_template_context_extract( true );
+		}
+
+		return $this->template;
+	}
+
+	/**
+	 * Adds list of Templates to the Tickets Emails settings tab.
+	 * 
+	 * @since TBD
+	 * 
+	 * @param  [] $fields Current array of Tickets Emails settings fields.
```

Comment:
I think the nomenclature we're using to define parameters is different.

```suggestion
	 * @param  array $fields Current array of Tickets Emails settings fields.
```